osu.Game.Tests/Visual/SongSelectV2/TestSceneSongSelect.cs

```diff
@@ -172,6 +174,100 @@ void decreaseModSpeed() => AddStep("decrease mod speed", () =>
             });
         }
 
+        [Test]
+        public void TestAutoplayShortcut()
+        {
+            ImportBeatmapForRuleset(0);
```

Comment:
As the `Hotkeys` region keeps growing, I'm beginning to think maybe it makes sense to split it out to its own test scene (as a general alternative to defining regions in this test)